src/layouts/product-integration-layout/index.tsx

```diff
@@ -126,18 +127,102 @@ export default function ProductIntegrationLayout({
 					<meta name="robots" content="noindex, nofollow" />
 				</HashiHead>
 			)}
-			<Header
-				productSlug={currentProduct.slug}
+
+			<ContentHeaderCard
 				className={s.header}
-				integration={integration}
-				activeRelease={activeRelease}
-				getVersionChangedURL={getVersionChangedURL}
-				onInstallClicked={() => {
-					console.log('TODO, probably remove this')
-				}}
+				icon={currentProduct.slug !== 'sentinel' ? currentProduct.slug : null}
+				title={integration.name}
+				attribution={`@${integration.organization.slug}`}
+				description={integration.description}
+				note={lastUpdatedString(integration.updated_at)}
+				badges={getIntegrationBadges(integration, true)}
+				dropdown={
+					integration.versions.length > 1
+						? {
+								text: versionString(
+									activeRelease.version,
+									integration.versions
+								),
+								items: integration.versions
+									.filter((e: string) => e !== activeRelease.version)
+									.map((version: string) => {
+										return {
+											text: versionString(version, integration.versions),
+											href: getVersionChangedURL(version),
+										}
+									}),
+						  }
+						: undefined
+				}
+				links={[
+					{
+						text: 'GitHub',
+						href: integration.subdirectory
+							? `${integration.repo_url}/tree/main${integration.subdirectory}`
+							: integration.repo_url,
```

Comment:
🙋🏻 📌 Maybe out of scope for this PR -- how do we know this is the subdirectory URL? It assumes `main` is the branch, but what if the repo the integration comes from doesn't have a `main` branch?

Reply:
This is a great callout!  I will have to think about this.

It's not quite a problem _yet_ but this will be something we have to address.  I'm going to file a ticket.

Reply:
🎟️ [Edge case: Link to an integration not on the `main` branch](https://app.asana.com/0/1203792616809419/1204110003219358)